Runtime/Data/Collection/List/ObservableList.cs

```diff
@@ -10,12 +10,27 @@
     using Zinnia.Extension;
     using Zinnia.Data.Type;
 
+    /// <summary>
+    /// The basis for all Observable List types.
+    /// </summary>
+    public abstract class ObservableList : MonoBehaviour
```

Comment:
Remove everything in this class' body - the element type is retrieved via reflection with the other suggestions now!